packages/cockpit/src/transactions/metrics/result.js

```diff
@@ -58,34 +60,41 @@ const getNamedWeekdays = map(evolve({
   key: weekdayNumber => getLocalizedWeekdayName(moment().day(weekdayNumber)),
 }))
 
-const buildDefaultWeekdays = map(
-  applySpec({
-    key: identity,
-    doc_count: always(0),
-  }),
+const buildDefaultWeekdays = reduce(
+  (acc, weekdayNumber) => mergeRight({ [weekdayNumber]: 0 }, acc),
```

Comment:
Aqui pode ser utilizado `Object Spread Operator`  ex
```suggestion
  (acc, weekdayNumber) => ({ [weekdayNumber]: 0 , ...acc }),
```

Reply:
Feito!